pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx

```diff
@@ -362,30 +351,52 @@ export function SpaceConfiguration({ creation = false }: { creation?: boolean; }
-				currentSpaceId != null ? (
-					<p>Current space ID: <span className='selectable-all'>{ currentSpaceId }</span></p>
-				) : (
-					<p>Currently in a personal space</p>
-				)
-			}
-			{ configurableElements }
-			<div className='input-container'>
-				<label>Features (cannot be changed after creation)</label>
-				<ul>
-					{
-						SPACE_FEATURES
-							.filter((feature) => currentConfig.features.includes(feature.id))
-							.map((feature) => (
-								<li key={ feature.id }>{ feature.name }</li>
-							))
-					}
-				</ul>
-			</div>
-			{ canEdit && <Button className='fill-x' onClick={ () => UpdateSpace(directoryConnector, modifiedData, () => navigate('/room')) }>Update space</Button> }
-			{ !canEdit && <Button className='fill-x' onClick={ () => navigate('/room') }>Back</Button> }
-			{ canEdit && currentSpaceId != null && <SpaceInvites spaceId={ currentSpaceId } /> }
+			<TabContainer className='flex-1'>
+				<Tab name='General'>
+					<div className='spaceConfigurationScreen-tab'>
+						<br />
+						{ configurableElements }
+						<div className='input-container'>
+							<label>Features (cannot be changed after creation)</label>
+							<ul>
+								{
+									SPACE_FEATURES
+										.filter((feature) => currentConfig.features.includes(feature.id))
+										.map((feature) => (
+											<li key={ feature.id }>{ feature.name }</li>
+										))
+								}
+							</ul>
+						</div>
+						{ canEdit && <Button className='fill-x' onClick={ () => UpdateSpace(directoryConnector, modifiedData, () => navigate('/room')) }>Update space</Button> }
+						{ !canEdit && <Button className='fill-x' onClick={ () => navigate('/room') }>Back</Button> }
+					</div>
+				</Tab>
+				<Tab name='Visitor Management'>
+					<div className='spaceConfigurationScreen-tab'>
+						<br />
+						<FieldsetToggle legend='Permission lists'>
+							<div className='input-container'>
+								<label>Admins</label>
+								<NumberListArea values={ currentConfig.admin } setValues={ (admin) => setModifiedData({ admin }) } readOnly={ !canEdit } />
+							</div>
+							<div className='input-container'>
+								<label>Banned users</label>
+								<NumberListArea values={ currentConfig.banned } setValues={ (banned) => setModifiedData({ banned }) } readOnly={ !canEdit } invalid={ invalidBans } />
+							</div>
+							<div className='input-container'>
+								<label>Allowed users</label>
+								<NumberListArea values={ currentConfig.allow } setValues={ (allow) => setModifiedData({ allow }) } readOnly={ !canEdit } invalid={ invalidAllow } />
+							</div>
+						</FieldsetToggle>
+						{ canEdit && currentSpaceId != null && <SpaceInvites spaceId={ currentSpaceId } /> }
+						<br />
+						{ canEdit && <Button className='fill-x' onClick={ () => UpdateSpace(directoryConnector, modifiedData, () => navigate('/room')) }>Update space</Button> }
+						{ !canEdit && <Button className='fill-x' onClick={ () => navigate('/room') }>Back</Button> }
```

Comment:
I think one back button is enough and this one can be removed

Reply:
Sure~

Reply:
Done